doc/mapbender_terminal_commands.md

```diff
@@ -0,0 +1,315 @@
+# Mapbender terminal commands
+Documentation of the Mapbender terminal commands needed for quick installation and updating within Composer.json use.
+
+Since Mapbender has implemented the composer tool, you can do all Composer editing from the console as well. The following commands are a selection that can be done via the console.
+
+If you look at all the commands and need the description, you can go to the page [Doku](https://getcomposer.org/doc/03-cli.md).
+
+# Creating a default composer.json
+The command `init-example` is used to create a composer.json, which is extracted from the bootstrap [Configuration](https://github.com/mapbender/mapbender-starter/blob/release/3.0.6/bootstrap). It is made an adjustment and overwrites the old composer.json a change and re-created. The composer.json is then created in the working directory.
```

Comment:
... Adjustments are made, and the old composer.json is being overwritten and re-created in the working directory.

Reply:
OK